engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/builtins/InvokeBuiltinMethodViaInteropTest.java

```diff
@@ -0,0 +1,151 @@
+package org.enso.interpreter.test.builtins;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+
+import com.oracle.truffle.api.interop.InteropLibrary;
+import org.enso.test.utils.ContextUtils;
+import org.graalvm.polyglot.Context;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ * This test tries to invoke some builtin methods on builtin types via the {@link
+ * com.oracle.truffle.api.interop.InteropLibrary interop} protocol.
+ */
+public class InvokeBuiltinMethodViaInteropTest {
+  private static Context ctx;
+
+  @BeforeClass
+  public static void setUp() {
+    ctx = ContextUtils.createDefaultContext();
+  }
+
+  @AfterClass
+  public static void tearDown() {
+    ctx.close();
+    ctx = null;
+  }
+
+  @Test
+  public void invokeGetMethodOnRef() {
+    var code =
+        """
+        import Standard.Base.Runtime.Ref.Ref
+
+        main = Ref.new 42
+        """;
+    var ref = ContextUtils.evalModule(ctx, code);
+    ContextUtils.executeInContext(
+        ctx,
+        () -> {
+          var interop = InteropLibrary.getUncached();
+          var refUnwrapped = ContextUtils.unwrapValue(ctx, ref);
+          assertThat(
+              "Ref builtin object should not have any members",
+              interop.hasMembers(refUnwrapped),
+              is(false));
+          assertThat(
+              "Ref should have a meta-object (Ref type)",
+              interop.hasMetaObject(refUnwrapped),
+              is(true));
+          var refMeta = interop.getMetaObject(refUnwrapped);
+          assertThat(
+              "Ref meta-object should have a 'get' method",
+              interop.isMemberInvocable(refMeta, "get"),
+              is(true));
+          var res = interop.invokeMember(refMeta, "get", new Object[] {refUnwrapped});
```

Comment:
The method can be **invoked as static** method on the `Ref` type with first argument being the `Ref` object.